src/query/builder.js

```diff
@@ -45,6 +45,22 @@ function Builder(client) {
 }
 inherits(Builder, EventEmitter);
 
+const validateWithArgs = function(alias, statement, method) {
+  if (typeof alias !== 'string') {
+    throw new Error(method + '() first argument must be a string');
```

Comment:
why not use template string?

Reply:
I was simply refactoring this to be used in both `with*` methods. I've noticed template aren't used in most of this codebase and I assumed this was support older JS interpreters. If you're saying template strings are fine I'll update.